pkgs/by-name/is/ism7mqtt/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+}:
+
+buildDotnetModule rec {
+  pname = "ism7mqtt";
+  version = "0.0.17";
+
+  src = fetchFromGitHub {
+    owner = "zivillian";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hswtuIJqpPCnRNHQ0Uq2NtKY06dERSmc4+nB8BKUjao=";
+  };
+
+  nugetDeps = ./deps.json;
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  enableParallelBuilding = false;
```

Comment:
```suggestion

```
redundant I think

Reply:
Is parallel building disabled by default? When testing the build, it would fail without this option set.

Reply:
Ah okay. It may not be. You can leave it.